bot/admin/admin.py

```diff
@@ -251,6 +275,30 @@ def parse_pastebin_link(url: str) -> str:
     return requests.get(url).text
 
 
+def _create_cogs_embed_string(loaded_cogs: Mapping[str, commands.Cog]) -> str:
+    """Method for creating the string used in the cogs embed.
+
+    Builds a string containing a list of all available Cogs. Each entry has an emoji representing if a Cog is currently
+    loaded or not.
+
+    Args:
+        loaded_cogs (Mapping[str, commands.Cog]): A Mapping containing all currently loaded Cogs.
+
+    Returns:
+        str: String containing the list of all Cogs and their current status.
+    """
+    string = ""
+
+    for cog in constants.INITIAL_EXTNS.keys():
+        if cog in loaded_cogs.keys():
+            string += constants.EMOJI_AVAILABLE
+        else:
+            string += constants.EMOJI_UNAVAILABLE
+        string += " --> {0}\n".format(cog[:-3])
+
+    return string
```

Comment:
You could do a more functional approach to this method to make it shorter: 
```
emojilist = list(map(lambda key: constants.EMOJI_AVAILABLE + ' -->' + key[:-3] if key in loaded_cogs.keys() else constants.EMOJI_UNAVAILABLE + ' -->' + key[:-3]))
return '\n'.join(emojilist)
```

Reply:
This isn't really shorter in my opinion because there is a lot going on in the first line. We can talk about changing this method to a mor functional approach but for now I'm gonna leave it as is. We can later talk about if this approach would be better.